src/engraving/rendering/score/segmentlayout.cpp

```diff
@@ -231,3 +233,84 @@ void SegmentLayout::layoutChordsStem(const Segment& segment, track_idx_t startTr
         }
     }
 }
+
+void SegmentLayout::placeParentheses(const Segment* segment, staff_idx_t staffIdx)
```

Comment:
Also, just for organization, it's a bit weird to have this in SegmentLayout, given that this is stricly related to all the rest of the courtesy layout code. So I'd rather put this in MeasureLayout together with all the rest. Or, if you want or think it could make sense, even create a new layout class like CourtesyLayout and put everything in there that relates to courtesies